simpleui-server/src/simpleui-server.ts

```diff
@@ -352,6 +364,62 @@ export class SimpleUIServer {
                 }
             });
 
+
+            // ------------------------------
+            // Handler for mock cmd requests
+            // ------------------------------
+            // Support path # /APP_NAME/UI_PROP/TAB_NAME/mock/cmd
+            const mockCmdQuery = [
+                `/:appName/:propsStub/:tabName/mock/cmd`
+            ];
+            displayUrl = `http://${os.hostname()}${webPortString}${mockCmdQuery[0]}`;
+            spacer1 = ' '.repeat(Math.max((105 - displayUrl.length), 1));
+            Logger.log(LogLevel.INFO, `Starting listener for ${displayUrl}${spacer1}(commands)`);
+            app.post(mockCmdQuery, async (req, res) => {
+                Logger.log(LogLevel.VERBOSE, `mock cmd request callback: ${++SimpleUIServer.requestCallbacks}`);
+                try {
+                    const cmdFilepath = req.query.file;
+                    Logger.log(LogLevel.DEBUG, `mock cmd filepath: ${cmdFilepath}`);
+                    SimpleUIServer.newMockDataURL = cmdFilepath;
+                } catch (err) {
+                    Logger.log(LogLevel.ERROR, err)
+                    Logger.log(LogLevel.ERROR, 'Error in mock cmd handler')
+                }
+                res.json({'mock data placeholder': 123})
+                /*
+                try {
+                    const props = PropsFileReader.getProps(
+                        `${req.params.propsStub}.properties`,
+                        `${req.params.appName}`, cmdVars.webPort);
+                    await SuiData.suiCommandRequest(req, res, props);
+                } catch (err) {
+
+                    const cmd = SuiData.getCmdFromReq(req);
+                    ServerUtil.logRequestDetails(LogLevel.ERROR, req,
+                        `Err in cmd request: ${err}`,
+                        'main cmd handler', '/mock/cmd', cmd);
+                }
+                */
+            });
+
+
+            // -----------------------------
+            // Handler for css_elements_to_json.php call
+            // -----------------------------
+            displayUrl = 'localhost:4100/bms/php/css_elements_to_json';
+            app.get('/bms/php/css_elements_to_json', async (req, res) => {
```

Comment:
I don't think `/bms/php/css_elements_to_json` is the right path to use for this handler.
I've implemented an alternative handler.